qc2md.py

```diff
@@ -330,6 +348,17 @@ def main():
         )
     )
 
+    standalone_categories = (
+        args.standalone_categories
+        if args.standalone_categories is not None
```

Comment:
This check is technically not important, as we already set a default value for `standalone_categories`. Also, an empty list (which is what `--standalone-categories -- input.txt` would result in) will not be `None`.